src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/time.css.twig

```diff
@@ -12,83 +12,81 @@
 
 /* Legend */
 
-.sf-profiler-timeline .legends .default { border-color: var(--color-default); }
-.sf-profiler-timeline .legends .section { border-color: var(--color-section); }
-.sf-profiler-timeline .legends .event-listener { border-color: var(--color-event-listener); }
-.sf-profiler-timeline .legends .template { border-color: var(--color-template); }
-.sf-profiler-timeline .legends .doctrine { border-color: var(--color-doctrine); }
-.sf-profiler-timeline .legends .messenger-middleware { border-color: var(--color-messenger-middleware); }
-.sf-profiler-timeline .legends .controller-argument-value-resolver { border-color: var(--color-controller-argument-value-resolver); }
+.sf-profiler-timeline .legends .timeline-default { border-color: var(--color-default); }
+.sf-profiler-timeline .legends .timeline-section { border-color: var(--color-section); }
+.sf-profiler-timeline .legends .timeline-event-listener { border-color: var(--color-event-listener); }
+.sf-profiler-timeline .legends .timeline-template { border-color: var(--color-template); }
+.sf-profiler-timeline .legends .timeline-doctrine { border-color: var(--color-doctrine); }
+.sf-profiler-timeline .legends .timeline-messenger-middleware { border-color: var(--color-messenger-middleware); }
+.sf-profiler-timeline .legends .timeline-controller-argument-value-resolver { border-color: var(--color-controller-argument-value-resolver); }
 
-svg {
+.timeline-graph {
```

Comment:
sf prefix.

Reply:
no needed. That's not part of the toolbar styles. the webprofiler UI does not conflict with anything in the app, and already avoids that prefix.